lunavl/sdk/estimators/face_estimators/credibility_check.py

```diff
@@ -0,0 +1,86 @@
+"""
+Module for estimate a credibility check from warped image.
+"""
+from FaceEngine import ICredibilityCheckEstimatorPtr
+from FaceEngine import CredibilityCheckEstimation
+
+from lunavl.sdk.errors.errors import LunaVLError
+from lunavl.sdk.errors.exceptions import CoreExceptionWrap, LunaSDKException
+from ..base import BaseEstimator
+from ..face_estimators.facewarper import FaceWarp, FaceWarpedImage
+from ...base import BaseEstimation
+
+from typing import Union, Dict, Any
+
+
+class CredibilityCheck(BaseEstimation):
+    """
+    Structure credibility check
+
+    Estimation properties:
+
+        - credibility_check
+    """
+
+    #  pylint: disable=W0235
+    def __init__(self, credibilityCheck: CredibilityCheckEstimation):
+        """
+        Init.
+
+        Args:
+            credibilityCheck: estimated credibility check
+        """
+        super().__init__(credibilityCheck)
```

Comment:
why `super`?

Reply:
Why should I remove the init statement? It sets the _coreEstimation, which is used to get the estimated value.

Reply:
fixed